dpa_check/dpa_check.py

```diff
@@ -43,6 +43,23 @@ def calc_model(model_spec, states, start, stop, T_dpa=None, T_dpa_times=None,
     model.comp['sim_z'].set_data(states['simpos'], times)
     model.comp['eclipse'].set_data(False)
     model.comp['1dpamzt'].set_data(T_dpa, T_dpa_times)
+
+    # Update to initialize the dpa0 pseudo-node. If 1dpamzt
+    # has an initial value (T_dpa) - which it does at
+    # prediction time (gets it from state0), then T_dpa0 
+    # is set to that.  If we are running the validation,
+    # T_dpa is set to None so we use the dvals in model.comp
+    #
+    # NOTE: If you change the name of the dpa0 pseudo node you
+    #       have to edit the new name into the if statement
+    #       below.
+    if 'dpa0' in model.comp:
+        if T_dpa is None:
+            T_dpa0 = model.comp["1dpamzt"].dvals
+        else:
+            T_dpa0 = T_dpa
+        model.comp['dpa0'].set_data(T_dpa0, T_dpa_times)
```

Comment:
If T_dpa is None, could T_dpa_times be None as well at this point?

Reply:
@jeanconn it actually is--thanks for pointing this out. For some reason this setup works fine for this and the PSMC model also. The goal here is to set the initial condition for the temperature. The `None` only comes in for the validation step, not prediction.

This is something that should probably be refactored in both models so that it's clearer what's happening here--but since this appears to be working now and only affects validation anyway, I would venture it's safe to leave it as-is for now?

Reply:
Gotcha.  Thanks for the clarification that it is only for validation anyway.  I should have asked about that too.  The mismatch (explicitly handling T_dpa None but not T_dpa_times None) just caught my eye, but if all the testing for predictions shows it has no impact I don't have an issue with it.